test/lyapunov.jl

```diff
@@ -7,14 +7,28 @@ using Test
 const _order = 28
 const _abstol = 1.0E-20
 
+#Lorenz system parameters
+const σ = 16.0
+const β = 4.0
+const ρ = 45.92
+
+#Lorenz system ODE:
+function lorenz!(t, x, dx)
+    dx[1] = σ*(x[2]-x[1])
+    dx[2] = x[1]*(ρ-x[3])-x[2]
+    dx[3] = x[1]*x[2]-β*x[3]
+    nothing
+end
+
+#Lorenz system Jacobian (in-place):
+function lorenz_jac!(jac, t, x)
```

Comment:
In the following lines, zero(x[1]) is preferred over 0.0*x[1], though this should be tested.